# Aliengo机器人硬件接口的配置文件
legged_unitree_hw:
  # 控制循环的频率 (Hz)，比A1高
  loop_frequency: 800
  # 控制周期的最大允许误差 (s)
  cycle_time_error_threshold: 0.002
  # 硬件控制线程的优先级 (0-99)
  thread_priority: 95
  # 电机功率限制，比A1高
  power_limit: 7
  # 足底接触力阈值 (N)
  contact_threshold: 40
